index.html

```diff
@@ -0,0 +1,162 @@
+          <li><i class="fas fa-store"></i>Dégustez au restaurant</li>
+        </ol>
+      </section>
+
+      <!--     RESTAURANTS     -->
+
+      <section id="restaurants">
+        <h2>Restaurants</h2>
+
+        <!--      PALETTE DU GOUT     -->
+
+        <article>
+          <figure>
+            <a href="./pages/la-palette-du-gout.html">
+              <span>Nouveau</span>
+              <img
+                src="images/restaurants/jay-wennington-N_Y88TWmGwA-unsplash.jpg"
+                alt="plat fin gastronomique et deux verres de vin"
+              />
+            </a>
+            <figcaption>
+              <a href="./pages/la-palette-du-gout.html">
+                <h3>La palette du goût</h3>
+                <p>Ménilmontant</p>
+              </a>
+              <i class="far fa-heart"></i>
+            </figcaption>
+          </figure>
+        </article>
+
+        <!--     NOTE ENCHANTEE     -->
+
+        <article>
+          <figure>
+            <a href="./pages/la-note-enchantee.html">
+              <span>Nouveau</span>
+              <img
+                src="images/restaurants/stil-u2Lp8tXIcjw-unsplash.jpg"
+                alt="plat en sauce rouge avec salade composée"
+              />
+            </a>
+            <figcaption>
+              <a href="./pages/la-note-enchantee.html">
+                <h3>La note enchantée</h3>
+                <p>Charonne</p>
+              </a>
+              <i class="far fa-heart"></i>
+            </figcaption>
+          </figure>
+        </article>
+
+        <!--     A LA FRANCAISE     -->
+
+        <article>
+          <figure>
+            <a href="./pages/a-la-francaise.html">
+              <img
+                src="images/restaurants/toa-heftiba-DQKerTsQwi0-unsplash.jpg"
+                alt="toast agrémenté de verdure et mozzarella"
+              />
+            </a>
+            <figcaption>
+              <a href="./pages/a-la-francaise.html">
+                <h3>À la française</h3>
+                <p>Cité rouge</p>
+              </a>
+              <i class="far fa-heart"></i>
+            </figcaption>
+          </figure>
+        </article>
+
+        <!--     DELICE DES SENS     -->
+
+        <article>
+          <figure>
+            <a href="./pages/le-delice-des-sens.html">
+              <img
+                src="images/restaurants/louis-hansel-shotsoflouis-qNBGVyOCY8Q-unsplash.jpg"
+                alt="plat à base de poisson dans une assiette en forme de tuile"
+              />
+            </a>
+            <figcaption>
+              <a href="./pages/le-delice-des-sens.html">
+                <h3>Le délice des sens</h3>
+                <p>Folie-Méricourt</p>
+              </a>
+              <i class="far fa-heart"></i>
+            </figcaption>
+          </figure>
+        </article>
+      </section>
+    </main>
+
+    <!--     FOOTER     -->
+
+    <footer>
+      <div>
+        <h4>ohmyfood</h4>
```

Comment:
Don't sure of h4

Reply:
h3 then ? as I'm not sure that this title is as important as others sections titles...